### ISSUE TITLE
runAutoEncoderDenoising "no applicable method for 'compile'" error.

### ISSUE THREAD

**Issue by wlwang20** [theMILOlab/SPATA2#83]
Hi- I am currently attempting to try the Autoencoder denoising. I've already installed tensorflow and keras in R following the [following instructions provided here](https://tensorflow.rstudio.com/install/). I've checked that both tensorflow and keras are loaded in correctly using simple commands: `tf$constant("Hello Tensorflow!")` yields `tf.Tensor(b'Hello Tensorflow!', shape=(), dtype=string)` and `k_backend()` yields `"tensorflow".` However, when I run the commands below, I keep running into an error that states `no applicable method for 'compile'.` Please see screenshot below for the full error message, and let me know how I can do to ameliorate it. 

object_275_T <- downloadSpataObject("275_T")
object_275_T <-
    runAutoencoderDenoising(
        object = object_275_T, 
        activation = "selu", 
        bottleneck = 56, 
        epochs = 20, 
        layers = c(128, 64, 32), 
        dropout = 0.1
    )

<img width="630" alt="Screenshot 2023-07-19 at 9 33 13 AM" src="https://github.com/theMILOlab/SPATA2/assets/43974296/4a7db9c9-6117-491a-8405-0e8a00f658d3">

**Comment by yuyzha** [comment]
Hello, I encountered the same error. Have you solved the problem and could you please share your solutions?
